README.md

```diff
@@ -18,9 +18,18 @@ npm install eslint-teamcity --save-dev
 ```
 
 ## Usage
+* as onrun eslint formatter:
 ```sh
   eslint --format './node_modules/eslint-teamcity/index.js' myfiletolint.js
 ```
+* as post runnig formatter, work with standart eslint json report
+```sh
+  node index.js result.json
```

Comment:
I'm not following. Say, as an example, I have a new project and I've added eslint-teamcity to my project. If I wanted to run this command to parse an ESLint JSON file, I would want to run this command. However, when I run it, it would look for `index.js` in my project folder (which wouldn't exist), instead of the `index.js` from eslint-teamcity. We want to make sure that we're running the correct `index.js` file so I think the command for your example would probably be: `node node_modules/eslint-teamcity/index.js result.json`

I don't think that people would want to navigate to the eslint-teamcity folder to directly run the command 😄